src/editor/pages/api/token/delete.ts

```diff
@@ -0,0 +1,9 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+import * as FlooqApi from '../../../helper/flooq-api'
+
+const handler = async ( req: NextApiRequest, res: NextApiResponse ): Promise<void> => {
+  const response = await FlooqApi.deleteRequest( req, `/api/Token/${JSON.parse( req.body ).id}` )
```

Comment:
this is a bit ugly. Instead of passing the id in the body you can use a query param. Then you don't have to parse the body before accessing the id.